analyzers/src/SonarAnalyzer.CSharp/Rules/StringLiteralShouldNotBeDuplicated.cs

```diff
@@ -40,15 +41,15 @@ public sealed class StringLiteralShouldNotBeDuplicated : StringLiteralShouldNotB
             SyntaxKind.StructDeclaration,
             SyntaxKindEx.RecordClassDeclaration,
             SyntaxKindEx.RecordStructDeclaration,
-            SyntaxKind.CompilationUnit,
+            SyntaxKind.CompilationUnit
```

Comment:
Why do you remove trailing commas? These are good for line copy&paste and reording.

Reply:
They are not needed. I think we don't modify the code that much and we can add them later, if needed.